doc/rbd/iscsi-target-ansible.rst

```diff
@@ -146,11 +103,16 @@ On the Ansible installer node, perform the following steps.
    ::
 
        # cd /usr/share/ceph-ansible
-       # ansible-playbook ceph-iscsi-gw.yml
+       # ansible-playbook site.yml --limit iscsigws
 
    .. note::
-    The Ansible playbook will handle RPM dependencies, RBD creation
-    and Linux IO configuration.
+    The Ansible playbook will handle RPM dependencies, setting up daemons,
+    and installing gwcli so it can be used to create iSCSI targets and export
+    RBD images as LUs. In past versions, iscsigws.yml could define the iSCSI
```

Comment:
LUNs?

Reply:
I changed it to LUNs in the repush, so it makes sense to most people and is consistent with the usage in other places.